cla_public/templates/privacy.html

```diff
@@ -54,82 +54,53 @@ <h2>{{ _('How is my privacy protected?') }}</h2>
-    only be to prevent fraud, help detect crime, if it is required by law or by a court order or If we are
+    only be to prevent fraud, to help detect crime, if it is required by law or by a court order or if we are
     very worried about your safety or the safety of others.{% endtrans %}
   </p>
 
-  <p>
-    {{ _('Read the full Civil Legal Advice Privacy Statement here.') }}
-  </p>
-
-
-  <h2{{ _('Civil Legal Advice Privacy Statement') }}></h2>
   <p>
     {% trans %}In order to provide a service to you, Civil Legal Advice (<abbr title="Civil Legal Advice">CLA</abbr>)
     may need to view, collect, record and hold (‘process’) personal data about you.{% endtrans %}
   </p>
   <p>
-    {% trans %}Personal data is information about you that can be used to identify you in some way
-    e.g. your name or telephone number. You can look at the
-    <a href="http://ico.org.uk/for_organisations/data_protection/the_guide/key_definitions#personal-data" rel="external">full legal definition</a>.{% endtrans %}
-  </p>
-  <p>
-    {% trans %}<abbr title="Civil Legal Advice">CLA</abbr> is committed to protecting your personal data and respecting your privacy.
-    We are legally bound by the Data Protection Act 1998 to ensure that we only use your personal
-    data for the reasons we tell you about. We must also hold it securely.{% endtrans %}
+    {% trans %}Personal data is information about you as an individual. It can be your name, address or telephone number. It can also include the information that you have provided in this form, such as your financial circumstances.{% endtrans %}
   </p>
+
   <p>
     {% trans %}This statement explains how <abbr>CLA</abbr> will process any personal data that we collect
     from you or that you provide to us.{% endtrans %}
   </p>
 
-
-  <h2{{ _('What is Civil Legal Advice?') }}></h2>
-  <p>
-    <abbr title="Civil Legal Advice">CLA</abbr> {{ _('is a national adviceline for England and Wales, paid for by Legal Aid.') }}
-  </p>
   <p>
     {% trans %}The Legal Aid Agency (<abbr title="The Legal Aid Agency">LAA</abbr>) is responsible for the <abbr>CLA</abbr>
-    service and the Ministry of Justice (<abbr title="The Ministry of Justice">MoJ</abbr>) is the Data Controller
-    for the purposes of the Data Protection Act 1998. They are responsible for determining the purposes for which
-    and the manner in which any personal data are, or are to be processed.{% endtrans %}
-  </p>
-  <p>
-    {% trans %}For a more comprehensive explanation see
-    <a href="http://ico.org.uk/for_organisations/data_protection/the_guide/key_definitions#personal-data" rel="external">this guide</a>.{% endtrans %}
+    service and the Ministry of Justice (<abbr title="The Ministry of Justice">MoJ</abbr>) is the Data Controller. The Legal Aid Agency (<abbr title="The Legal Aid Agency">LAA</abbr>) is an Executive Agency of the Ministry of Justice (<abbr title="The Ministry of Justice">MoJ</abbr>).{% endtrans %}
```

Comment:
That is my only feed back really. Just go through and make sure your lines have a more uniform length by splitting them up.